### ISSUE TITLE
React: Language selector - drop down is not operable.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4331]
## Detailed description
On iPhone voiceover

Issue:
In iPhone, the dropdown is identified by Voiceover. But it is not operable 

Expected:
Screen reader should be able to Identify the dropdown and the user should be able to navigate through the options and make selection

**Comment by kennylam** [comment]
@RobertaJHahn @Praveen-sr a PR has been opened for this issue https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/4335
